meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -22,6 +26,10 @@ do_compile_append_imx7s-warp-mbl() {
 	oe_runmake u-boot-dtb.imx
 }
 
-do_deploy_append() {
+do_deploy_append_imx7s-warp-mbl() {
```

Comment:
why can't u-boot-fslc do_deploy_append() be the same for both imx7s and imx7d?

Reply:
explained in git comments.